superset-frontend/src/components/Select/styles.tsx

```diff
@@ -241,9 +242,42 @@ export const DEFAULT_STYLES: PartialStylesConfig = {
   }),
 };
 
-const { ClearIndicator, DropdownIndicator, Option } = defaultComponents;
+const multiInputStyle = css`
+  padding: 0 6px;
+  vertical-align: middle;
+`;
 
-export const DEFAULT_COMPONENTS: SelectComponentsConfig<any> = {
+type SelectComponentsType = Omit<SelectComponentsConfig<any>, 'Input'> & {
+  Input: ComponentType<InputProps>;
+};
+
+// react-select is missing selectProps from their props type
+// so overwriting it here to avoid errors
+type InputProps = {
+  selectProps: {
+    isMulti: boolean;
+    value: {
+      length: number;
+    };
+    placeholder: string;
+  };
+  cx: (a: string | null, b: any, c: string) => string | void;
```

Comment:
i'd imagine we can pull `cx`, `className`, and `getStyles`attributes form a type in emotion or the superset styled library. Any thoughts @rusackas ?

Reply:
Most of these attribute types are pulled in from the existing type on InputProps from `react-select` with the exception that I'm adding `selectProps` which exists on the component, but the library doesn't have it present in the types.. a bug on their part I think. Not sure how removing any attributes would break anything down the road.

Reply:
Can you try if this works?

```typescript
import { InputProps as ReactSelectInputProps } from 'react-select';

type InputProps = ReactSelectInputProps & {
  placeholder?: string;
  selectProps?: CommonProps<any>['selectProps'];
}
```

Reply:
Ah, those were exported. I missed them the first time around. thx. I am admittedly still leveling up my typescript chops, so this sort of worked, but I still had to declare some times for the selectProps properties below. Not sure why they didn't import correctly off the interface. It could be because I'm taking the placeholder of selectProps and assigning it to the placeholder on the input and maybe the types aren't the same, but if you have any suggestions, lmk!